web_widget_image_download/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```

Comment:
Talking with @Yajo, he is right about the possibility of using this widget through https://github.com/OCA/website/tree/8.0/website_backend_views, and in this case, having an LGPL widget that can be reused in any frontend is a better option.